ooni/nettests/experimental/tls_handshake.py

```diff
@@ -743,6 +766,7 @@ def handshakeSucceeded(connection):
 
             self.report['host'] = host
             self.report['port'] = port
+            self.report['sni'] = self.sni
```

Comment:
You probably want to just set this inside of the `setUp` method as it doesn't need to be changed across time. It's also more robust against possible failures of the test.